raiden/tests/scenarios/mfee3_only_imbalance_fees.yaml

```diff
@@ -36,8 +36,9 @@ nodes:
     proportional-imbalance-fee:
       - "0x62083c80353Df771426D209eF578619EE68D5C7A"
       - 10_000  # 1% imbalance fee on channel balance
+    no-cap-mediation-fees: true
 
-## This is the MFEE3 scenario. It creates a network with topology 0 -> 1 -> 2 -> 3 and checks
+## This is the MFEE3 scenario. It creates a network with topology 0 -> 1 -> 2 and checks
```

Comment:
Can we align that description with what you have written in the Readme
``` 
Make a transfer over a single mediator with enabled imbalance fees. The
channels start at maximum imbalance, which causes negative fees. To test this,
the mediator fee capping has been turned off.
```

Reply:
Do we generally want to duplicate the text in the README and the scenario, or should we skip one in favor of the other?

Reply:
In general, we want to have the same scenario description in the Readme and in the scenarios. Does that make sense or do you see a reason why those should differ?

Reply:
It's only the usual aversion of duplication content (will get out of sync, harder to maintain). I'll copy it for now.